tuftool/tests/delegation_commands.rs

```diff
@@ -1280,16 +1267,14 @@ fn remove_role_recursive_command() {
         .success();
 
     // Load the updated repo
-    let temp_datastore = TempDir::new().unwrap();
-    let updated_metadata_base_url = &test_utils::dir_url(new_repo_dir.path().join("metadata"));
-    let updated_targets_base_url = &test_utils::dir_url(new_repo_dir.path().join("targets"));
+    let updated_metadata_base_url = &dir_url(new_repo_dir.path().join("metadata"));
     let repo = Repository::load(
         &tough::FilesystemTransport,
         Settings {
             root: File::open(&root_json).unwrap(),
-            datastore: temp_datastore.as_ref(),
-            metadata_base_url: updated_metadata_base_url,
-            targets_base_url: updated_targets_base_url,
+            datastore: None,
+            metadata_base_url: updated_metadata_base_url.clone(),
+            targets_base_url: dir_url(new_repo_dir.path().join("targets")),
```

Comment:
nit: why do metadata URL above and targets URL here?

Reply:
It's used later in a `tuftool update` call on line 1360.

Reply:
I know, I guess I didn't word my question well.  I meant, why not do targets URL the same way, so this doesn't look different?  I was searching around for reasons, where if it looked consistent I think the purpose would be more obvious.  Still, just a nit.